RecoHI/HiEgammaAlgos/python/AODHIPhotonProducer_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+AODHIPhotonProducer = cms.EDProducer(
```

Comment:
start instances with lower case letters (and keep the file name matching the module instance name)